src/targets/services/fetchOpenPathTrips.js

```diff
@@ -42,41 +42,51 @@ const fetchOpenPathTrips = async () => {
 
   const client = CozyClient.fromEnv(process.env, { schema })
 
-  let account
+  let accounts
   if (user) {
     // Find account from given user token
-    account = await queryAccountByToken(client, user)
+    const account = await queryAccountByToken(client, user)
+    accounts = [account]
   } else {
-    // Find account from creation date
-    account = await queryLastServiceAccount(client)
+    // Find all service accounts
+    accounts = await queryServiceAccounts(client)
   }
 
-  if (!account) {
+  if (!accounts) {
```

Comment:
You should check with accounts.length because [] is a truthy value. So it will enter this condition.

Reply:
And be careful because line 48 if account is undefined, we wille have `accounts = [undefined]`, `[undefined].length` return 1